client/modules/User/components/SignupForm.jsx

```diff
@@ -9,49 +9,64 @@ import Button from '../../../common/Button';
 import apiClient from '../../../utils/apiClient';
 import useSyncFormTranslations from '../../../common/useSyncFormTranslations';
 
-function asyncValidate(fieldToValidate, value) {
+// Debounce utility function
```

Comment:
Ideally we should remove these comments, since a lot of it might feel self-explanatory from the code!